server/posts.tsx

```
@@ -0,0 +1,138 @@
+                tags: true,
+                user: true,
+                comments: true,
+            },
+        });
+
+        if (!post) {
+            throw new Error('Post not found');
+        }
+
+        return post;
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)